modules/aws/s3.go

```diff
@@ -260,6 +260,35 @@ func EmptyS3BucketE(t testing.TestingT, region string, name string) error {
 	return err
 }
 
+// GetS3BucketLoggingTarget fetches the given bucket's logging configuration status and returns it as a string
+func GetS3BucketLoggingTarget(t testing.TestingT, awsRegion string, bucket string) string {
+	loggingTarget, err := GetS3BucketLoggingTargetE(t, awsRegion, bucket)
+	require.NoError(t, err)
+
+	return loggingTarget
+}
+
+// GetS3BucketLoggingE fetches the given bucket's versioning configuration status and returns it as a string
+func GetS3BucketLoggingTargetE(t testing.TestingT, awsRegion string, bucket string) (string, error) {
+	s3Client, err := NewS3ClientE(t, awsRegion)
+	if err != nil {
+		return "", err
+	}
+
+	res, err := s3Client.GetBucketLogging(&s3.GetBucketLoggingInput{
+		Bucket: &bucket,
+	})
+	if err != nil {
+		return "", err
+	}
+
+	if res.LoggingEnabled != nil {
+		return aws.StringValue(res.LoggingEnabled.TargetBucket), nil
+	}
+
+	return "", err
```

Comment:
I missed this in the first pass, but I just realized that this won't return an error, but `nil` (because you had the `err` check before). It might be more favorable to return an error in this case, since logging is not enabled.

Can you make this return a custom error (e.g., `S3AccessLoggingNotEnabledErr`)?

Reply:
Good shout!